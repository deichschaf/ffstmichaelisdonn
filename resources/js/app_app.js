import React from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import AnwesenheitOverview from './app/anwesenheit/Overview';
import EinsaetzeEditor from './app/einsaetze/editor';
import PageAppContainer from './component/organisms/PageAppContainer';
import MoveUp from './component/useHooks/MoveUp';
import ErrorBoundary from './component/organisms/errorboundary';
import AppTacticalMonitor from './app/tacticalmonitor';

export default function App() {
  const path = '/app';
  return (
    <ErrorBoundary>
      <BrowserRouter forceRefresh={true}>
        <MoveUp />
        <PageAppContainer>
          <ErrorBoundary>
            <Routes>
              <Route path={`${path}/einsaetze`} element={<EinsaetzeEditor />} />
              <Route path={`${path}/anwesenheit`} element={<AnwesenheitOverview />} />
              <Route path={`${path}/tacticalmonitor`} element={<AppTacticalMonitor />} />
            </Routes>
          </ErrorBoundary>
        </PageAppContainer>
      </BrowserRouter>
    </ErrorBoundary>
  );
}
