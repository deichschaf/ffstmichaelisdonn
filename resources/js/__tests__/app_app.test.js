// Auto-generated do not edit

/* eslint-disable import/no-extraneous-dependencies */
/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import app_app from '../app_app';

describe('app_app test', () => {
  it('app_app should match snapshot', () => {
    const component = renderer.create(<app_app />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
