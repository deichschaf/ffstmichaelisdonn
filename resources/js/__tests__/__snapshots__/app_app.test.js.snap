// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app_app test app_app should match snapshot 1`] = `<app_app />`;
